cirq-core/cirq/testing/lin_alg_utils.py

```diff
@@ -190,6 +190,16 @@ def assert_allclose_up_to_global_phase(
     # pylint: enable=unused-variable
 
     actual, desired = linalg.match_global_phase(actual, desired)
+
```

Comment:
please don't change this method... instead track the failing tests an modify them

Reply:
Sorry, I've removed the changes from that method and modified only the failing test. 
The idea of the modification is still the same:
```
Z**0 * X ** 0 * Z ** 0 == Z**-0.5* X ** 0 * Z ** +0.5
```